classes/utils/Utils.py

```diff
@@ -0,0 +1,321 @@
+    ENDER_DRAGON = (19, 'ender_dragon')
+    ENDERMAN = (20, 'enderman')
+    ENDERMITE = (21, 'endermite')
+    EVOKER_FANGS = (22, 'evoker_fangs')
+    EVOKER = (23, 'evoker')
+    EXPERIENCE_ORB = (24, 'experience_orb')
+    EYE_OF_ENDER = (25, 'eye_of_ender')
+    FALLING_BLOCK = (26, 'falling_block')
+    FIREWORK_ROCKET = (27, 'firework_rocket')
+    FOX = (28, 'fox')
+    GHAST = (29, 'ghast')
+    GIANT = (30, 'giant')
+    GUARDIAN = (31, 'guardian')
+    HORSE = (32, 'horse')
+    HUSK = (33, 'husk')
+    ILLUSIONER = (34, 'illusioner')
+    ITEM = (35, 'item')
+    ITEM_FRAME = (36, 'item_frame')
+    FIREBALL = (37, 'fireball')
+    LEASH_KNOT = (38, 'leash_knot')
+    LLAMA = (39, 'llama')
+    LLAMA_SPIT = (40, 'llama_spit')
+    MAGMA_CUBE = (41, 'magma_cube')
+    MINECART = (42, 'minecart')
+    CHEST_MINECART = (43, 'chest_minecart')
+    COMMAND_BLOCK_MINECART = (44, 'command_block_minecart')
+    FURNACE_MINECART = (45, 'furnace_minecart')
+    HOPPER_MINECART = (46, 'hopper_minecart')
+    SPAWNER_MINECART = (47, 'spawner_minecart')
+    TNT_MINECART = (48, 'tnt_minecart')
+    MULE = (49, 'mule')
+    MOOSHROOM = (50, 'mooshroom')
+    OCELOT = (51, 'ocelot')
+    PAINTING = (52, 'painting')
+    PANDA = (53, 'panda')
+    PARROT = (54, 'parrot')
+    PIG = (55, 'pig')
+    PUFFERFISH = (56, 'pufferfish')
+    ZOMBIE_PIGMAN = (57, 'zombie_pigman')
+    POLAR_BEAR = (58, 'polar_bear')
+    TNT = (59, 'tnt')
+    RABBIT = (60, 'rabbit')
+    SALMON = (61, 'salmon')
+    SHEEP = (62, 'sheep')
+    SHULKER = (63, 'shulker')
+    SHULKER_BULLET = (64, 'shulker_bullet')
+    SILVERFISH = (65, 'silverfish')
+    SKELETON = (66, 'skeleton')
+    SKELETON_HORSE = (67, 'skeleton_horse')
+    SLIME = (68, 'slime')
+    SMALL_FIREBALL = (69, 'small_fireball')
+    SNOW_GOLEM = (70, 'snow_golem')
+    SNOWBALL = (71, 'snowball')
+    SPECTRAL_ARROW = (72, 'spectral_arrow')
+    SPIDER = (73, 'spider')
+    SQUID = (74, 'squid')
+    STRAY = (75, 'stray')
+    TRADER_LLAMA = (76, 'trader_llama')
+    TROPICAL_FISH = (77, 'tropical_fish')
+    TURTLE = (78, 'turtle')
+    EGG = (79, 'egg')
+    ENDER_PEARL = (80, 'ender_pearl')
+    EXPERIENCE_BOTTLE = (81, 'experience_bottle')
+    POTION = (82, 'potion')
+    TRIDENT = (83, 'trident')
+    VEX = (84, 'vex')
+    VILLAGER = (85, 'villager')
+    IRON_GOLEM = (86, 'iron_golem')
+    VINDICATOR = (87, 'vindicator')
+    PILLAGER = (88, 'pillager')
+    WANDERING_TRADER = (89, 'wandering_trader')
+    WITCH = (90, 'witch')
+    WITHER = (91, 'wither')
+    WITHER_SKELETON = (92, 'wither_skeleton')
+    WITHER_SKULL = (93, 'wither_skull')
+    WOLF = (94, 'wolf')
+    ZOMBIE = (95, 'zombie')
+    ZOMBIE_HORSE = (96, 'zombie_horse')
+    ZOMBIE_VILLAGER = (97, 'zombie_villager')
+    PHANTOM = (98, 'phantom')
+    RAVAGER = (99, 'ravager')
+    LIGHTNING_BOLT = (100, 'lightning_bolt')
+    PLAYER = (101, 'player')
+    FISHING_BOBBER = (102, 'fishing_bobber')
```

Comment:
This looks wrong, where exactly did you get the information from? https://wiki.vg/index.php?title=Entity_metadata&diff=15700&oldid=15581 Appears to have different values

Reply:
Protocol for 1.15.2, I'll remove ids, we'll handle it later